src/vs/editor/contrib/suggest/suggestModel.ts

```diff
@@ -46,7 +46,19 @@ export class LineContext {
 		}
 		const pos = editor.getPosition();
 		model.tokenizeIfCheap(pos.lineNumber);
-		const word = model.getWordAtPosition(pos);
+
+		// When we are at an embedded language boundary, check the word at the previous character
```

Comment:
This some pretty ugly logic. Let  me know if there is some better way to express this